ui/src/containers/Layout.js

```diff
@@ -56,8 +61,22 @@ const Layout = (props) => {
     'data-cy-state-isexpanded': sidebar.expanded,
     actions: [
       {
-        label: intl.translate('alerts'),
+        label: intl.translate('dashboard'),
         icon: <i className="fas fa-desktop" />,
+        onClick: () => {
+          history.push('/dashboard');
```

Comment:
I think dashboard should be served on `/` and alerts should be served on `/alerts`. Also alerts screen should be removed from this side bar as it will be added in the navbar soon

Reply:
Absolutely!
When the navbar is ready, we can consider moving the alert page to there with some design changes too. 
I will open a ticket for this once we have the design from @Cuervino.